### ISSUE TITLE
localStorage.getItem() returns NULL for all keys - broken in v1.1.1 and above

### ISSUE THREAD

**Issue by acoutts** [aihornmac/mobx-localstorage#3]
``` javascript
import localStorageMobx from "mobx-localstorage"

console.log(localStorageMobx.getItem("foo") // null -- incorrect
console.log(localStorage.getItem("foo") // bar -- correct
```

For some reason, all values returned by mobx-localstorage are null but using the normal localstorage returns the correct value. Seems to have broken in v1.1.1, because it doesn't work in 1.1.1 or 1.1.2, but does work in 1.1.0.

**Comment by aihornmac** [comment]
@acoutts Thanks for your feedback!
I wonder if the value of `foo` is set with native localStorage in other places in the same page before you run `localStorageMobx.getItem`

**Comment by aihornmac** [comment]
@acoutts Never mind, I found the reason. Turns out that calling `super(entries)` isn't working correctly. I moved the process down after super. Fixed in `1.1.3`, please try!

**Comment by acoutts** [comment]
That did it! Fixed now - thanks a lot for all of your work on this library. It's very handy when working with localStorage and MobX!

I use it to listen/react to changes in localStorage done by a 3rd party charting library on the page. Very easy to just write a reaction and hook in.
